native/cocos/ui/edit-box/EditBox-ios.mm

```diff
@@ -546,7 +546,9 @@ - (id) createTextField:    (const cc::EditBox::ShowInfo*)showInfo {
     ((UITextField*)[ret inputOnToolbar]).text = [NSString stringWithUTF8String:showInfo->defaultValue.c_str()];
     ((UITextField*)[ret inputOnView]).text = [NSString stringWithUTF8String:showInfo->defaultValue.c_str()];
     setTextFieldReturnType((UITextField*)[ret inputOnToolbar], showInfo->confirmType);
+    setTextFieldReturnType((UITextField*)[ret inputOnView], showInfo->confirmType);
     setTextFieldKeyboardType((UITextField*)[ret inputOnToolbar], showInfo->inputType);
+    setTextFieldKeyboardType((UITextField*)[ret inputOnView], showInfo->inputType);
```

Comment:
For the current implementation, we divide TextField into two parts, `inputOnToolbar` (represents the part related to the keyboard) and `inputOnView`(represents the part displayed when the edit box is clicked). The cause of the inconsistent display issue is not setting the same keyboard for inputOnView and inputOnToolbar.

Reply:
The consequence is that the keyboard related `inputOnView` will be displayed first, and then the keyboard related `inputOnToolbar` will be displayed. When the keyboard settings are the same, there is no sense of dissonance.